src/main/java/com/se/homeworktwo/Sym.java

```diff
@@ -1,5 +1,6 @@
 package com.se.homeworktwo;
 
+import java.util.HashMap;
 import java.util.Map;
```

Comment:
Can you remove this import?

Reply:
Added import.java.util.* instead of two separate imports